xdr/transaction_envelope_test.go

```diff
@@ -85,6 +85,66 @@ func createTx() TransactionEnvelope {
 	}
 }
 
+func createV2Tx() TransactionEnvelope {
```

Comment:
nit: technically it's a `CondV2` transaction, not a V2 transaction :eyes: 
```suggestion
func createCondV2Tx() TransactionEnvelope {
```